src/components/Contacts/ContactDetails/ContactDetailsTab/People/ContactDetailsTabPeople.tsx

```diff
@@ -0,0 +1,146 @@
+              <ContactPersonPrimaryText variant="subtitle1">
+                {`- ${t('Primary')}`}
+              </ContactPersonPrimaryText>
+            ) : null}
+          </ContactPersonRowContainer>
+          {/* Phone Number */}
+          {person.primaryPhoneNumber !== null ? (
+            <ContactPersonRowContainer>
+              <ContactPersonIconContainer>
+                <Phone color="disabled" />
+              </ContactPersonIconContainer>
+              <Typography variant="subtitle1">
+                {person.primaryPhoneNumber?.number}
+              </Typography>
+              {person.primaryPhoneNumber?.location ? (
+                <Typography variant="caption">
+                  {` - ${person.primaryPhoneNumber.location}`}
+                </Typography>
+              ) : null}
+            </ContactPersonRowContainer>
+          ) : null}
+          {/* Email Section */}
+          {person.primaryEmailAddress !== null ? (
+            <ContactPersonRowContainer>
+              <ContactPersonIconContainer>
+                <Email color="disabled" />
+              </ContactPersonIconContainer>
+              <Typography variant="subtitle1">
+                {person.primaryEmailAddress?.email}
+              </Typography>
+            </ContactPersonRowContainer>
+          ) : null}
+          {/* Birthday Section */}
+          {person.birthdayDay !== null && person.birthdayMonth ? (
+            <ContactPersonRowContainer>
+              <ContactPersonIconContainer>
+                <Cake color="disabled" />
+              </ContactPersonIconContainer>
+              {/* TODO: Change to local format for different countries */}
+              <Typography variant="subtitle1">
+                {person.birthdayYear
+                  ? `${person.birthdayMonth}/${person.birthdayDay}/${person.birthdayYear}`
+                  : `${person.birthdayMonth}/${person.birthdayDay}/1900`}
+              </Typography>
+            </ContactPersonRowContainer>
+          ) : null}
+          {/* Anniversary Section */}
+          {person.anniversaryDay !== null && person.anniversaryMonth ? (
+            <ContactPersonRowContainer>
+              <ContactPersonIconContainer>
+                <RingIcon color="disabled" />
+              </ContactPersonIconContainer>
+              {/* TODO: Change to local format for different countries */}
+              <Typography variant="subtitle1">
+                {person.anniversaryYear
+                  ? `${person.anniversaryMonth}/${person.anniversaryDay}/${person.anniversaryYear}`
+                  : `${person.anniversaryMonth}/${person.anniversaryDay}/1900`}
+              </Typography>
+            </ContactPersonRowContainer>
+          ) : null}
+        </ContactPersonTextContainer>
+      </ContactPersonContainer>
+    );
+  };
+
+  return (
+    <>
+      {primaryPerson !== null
+        ? personView(primaryPerson as ContactPersonFragment)
+        : null}
```

Comment:
You don't need the cast. The type error you get here without it is complaining that you're passing `undefined` to a function whose argument doesn't allow `undefined`. Changing the null check to check for more falsy values (including `undefined`) seems to make it happy.
```suggestion
      {primaryPerson ? personView(primaryPerson) : null}
```